src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.StringJoiner;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+
+        SimpleDateFormat dateFormat = new SimpleDateFormat("dd.MM.yyyy");
```

Comment:
![image](https://github.com/mate-academy/jv-salary-info/assets/140905715/68d93a46-eded-4692-812a-0cfdf10f8810)

Reply:
It should also be `DateTimeFormatter`